endpointmanager/pkg/versionsoperatorparser/versionsoperatorparser_test.go

```diff
@@ -0,0 +1,95 @@
+package versionsoperatorparser
```

Comment:
I think this file should be called `versionsresponse_test.go` just so that it matches up to the file it's testing!